js/classes/entities/midground/chad.js

```diff
@@ -417,15 +414,25 @@ class Chad {
             }
         }
 
+
         if (GAME.user.jabbing) {
             this.action = "slicing";
         }
-
         if (this.isOnGround && !(GAME.user.movingRight || GAME.user.movingLeft)) {
             this.action = "idle";
+            if (GAME.user.jabbing) {
+                this.action = "slicingStill";
```

Comment:
Thank you for adding this!

Reply:
Thanks for fixing the animation @calebkrauter !

Reply:
Happy to!